ci/bower-dev.sh

```diff
@@ -7,6 +7,7 @@ mv bower.json bower.json.tmp
 git remote add upstream https://github.com/ManageIQ/ui-components.git
 git fetch upstream
 git checkout bower-dev
+rm -rf dist
```

Comment:
This will delete also any ancillary files like the README and LICENSE, which might have side effects?